openfisca_us/parameters/usda/snap/homeless_shelter_deduction.yaml

```diff
@@ -0,0 +1,22 @@
+description: SNAP Homeless Shelter Deduction Parameters
+
+deduction:
+  2021-01-01: 143
+  metadata:
+    unit: currency-USD
+
+eligibility:
```

Comment:
remove this and make it part of the logic (homelessness is household-level)

Reply:
Homelessness is defined at the person level per the law:
"A State agency may provide a standard homeless shelter deduction of $143 a month to households in which all members are homeless individuals but are not receiving free shelter throughout the month."
https://www.law.cornell.edu/cfr/text/7/273.9

I was under the impression it would be household level, but it seems the law included this pretty explicitly. What do you think?

Reply:
Let's simplify to household-level for now, can always make more complicated later. But I think the microdata would show homelessness at the household level, and it's reasonable to ask users at the household level for the household calculator.